pyproject.toml

```diff
@@ -2,6 +2,49 @@
 requires = [
     "setuptools>=42",
     "wheel",
-    "pybind11~=2.6",
+    "pybind11==2.12.0",
+    "tomli>=0.10; python_version<'3.11'",
 ]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "mapbox_earcut"
+version = "1.0.2"
+requires-python = ">=3.7"
+authors = [{name = "Samuel Kogler", email = "samuel.kogler@gmail.com"}]
+license =  {file = "LICENSE.md"}
+description = "Python bindings for the mapbox earcut C++ polygon triangulation library."
+dependencies = ["numpy>=1.19.0"]
+
+[project.urls]
+Source = "https://github.com/skogler/mapbox_earcut_python"
+CSource = "https://github.com/mapbox/earcut.hpp"
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[tool.setuptools]
+zip-safe = false
+include-package-data = true
+
+[project.optional-dependencies]
+test = ["pytest"]
+
+
+[tool.cibuildwheel]
+# install the `test` extra
+test-extras = ["test"]
+
+# Run the package tests using `pytest`
+# also test against pre-release Numpy
+# TODO : when numpy 2.0 releases this can be reduced to just one pytest
+test-command = ["pytest {package}/tests",
```

Comment:
does this work? the tests aren't in the sdist -- I had to put acopy of the test fijles in the conda recipe to get the tests to run.

really, they should be installed with the package -- they are small, and it's realy good to be able to directly test the installed distro with a compiled package like this.